tests/test_harvesters.py

```diff
@@ -138,3 +176,66 @@ def test_copernicus_sentinel_harvester(self):
                                                            username='test', password='TEST')
         self.assertIsInstance(harvester._current_crawler, crawlers.CopernicusOpenSearchAPICrawler)
         self.assertIsInstance(harvester._ingester, ingesters.CopernicusODataIngester)
+
+    def test_osisaf_harvester_extra_excludes(self):
+        """ extra excludes should have passed by the excludes as a list in configuration file.
+        Otherwise, accossiated error must be raised """
+        harvester = harvesters.OSISAFHarvester(urls=[''], max_fetcher_threads=1, max_db_threads=1,
+                                               excludes=['ease', '_sh_polstere', ])
+        self.assertListEqual(list(harvester._current_crawler.excludes),
+                             ['/thredds/', 'http', 'ease', '_sh_polstere'])
+        harvester = harvesters.OSISAFHarvester(urls=[''], max_fetcher_threads=1, max_db_threads=1)
+        self.assertListEqual(list(harvester._current_crawler.excludes), ['/thredds/', 'http', ])
```

Comment:
it's already a list, no need to explicitly convert it

Reply:
Done in the next commit.